lib/constructs/interface.js

```diff
@@ -213,9 +213,6 @@ class Interface {
             }
             this.indexedGetter = member;
           } else if (isNamed(member)) {
-            if (this.namedGetter) {
-              throw new Error(`${msg}duplicated named getter`);
-            }
```

Comment:
You could also change the condition to `if (this.hasOwnProperty(member))` in order to keep at least a check on duplicated member names, but I noticed this kind of check is not in place for regular attributes either, so i preferred to have the same behaviour for getters and setters as well i.e. if multiple members have the same name, simply use the last defined one and discard all the others.